DataLayer/CmuData/CmuData.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QList>
+
+#include "I_CmuUnit.h"
+
+class CmuData: I_CmuData
+{
+public:
+    CmuData(const unsigned char& numberOfCells);
+
+    I_CmuUnit getCmuUnit(const unsigned char& index);
+
+private: 
+    QList<I_CmuUnit*> cmuUnits_;
```

Comment:
Should I use a `QList<QScopedPointer<I_CmuUnit>> cmuUnits` or `QScopedArrayPointer<I_CmuUnit>`